code/game/objects/items/robot/robot_items.dm

```diff
@@ -867,13 +873,18 @@
 		return
 	. = ..()
 
+/// Parent proc for all apparatus, tells the player that they can alt-click to drop.
+/obj/item/borg/apparatus/examine(mob/user)
+	. = ..()
+	. += "<span class='notice'> <i>Alt+click</i> will drop the currently stored item. </span>"
```

Comment:
```suggestion
	. += "<span class='notice'><i>Alt-click</i> will drop the currently stored item.</span>"
```

Just consistent with how we have it everywhere else.

Reply:
I think I'd rather this be per apparatus honestly, just so we get more detailed messages than "item".

Reply:
Copy that.